README.md

```diff
@@ -23,13 +23,15 @@ Each folder inside packages/ is considered a workspace. To see a list of all wor
 
 # Setup
 
-1). Ensure using NODE Version 14 (v14.19.0)
+These steps are for an install on a Mac. The Windows instructions are [here](https://github.com/usdigitalresponse/usdr-gost/wiki/Setting-up-a-development-environment-on-Windows-(native)).
```

Comment:
Perhaps we should link to both the [native](https://github.com/usdigitalresponse/usdr-gost/wiki/Setting-up-a-development-environment-on-Windows-(native)) and [WSL](https://github.com/usdigitalresponse/usdr-gost/wiki/Setting-up-a-development-environment-on-Windows-(WSL)) wiki pages?